src/picongpu/include/particles/manipulators/FreeImpl.hpp

```diff
@@ -31,35 +32,138 @@ namespace particles
 namespace manipulators
 {
 
-template<typename T_Functor>
-struct FreeImpl
-{
+    /** generic manipulator to create user defined manipulators
+     *
+     * @tparam T_Functor user defined functor
```

Comment:
can you move the `@tparam` below the description pls? (Doxygen style is title - brief - params)